cli/jobs/pipelines/automl/image-classification-multiclass-pipeline.yml

```diff
@@ -0,0 +1,56 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
+type: pipeline
+
+description: Image multiclass classification using pipeline job
+
+experiment_name: pipeline_image_multiclass_classification
+
+settings:
+  continue_on_step_failure: false
+  default_compute: azureml:gpu-cluster
+
+inputs:
+  image_multiclass_classification_train_data:
+    type: mltable
+    path: ../../automl-standalone-jobs/cli-automl-image-classification-multiclass-task-fridge-items/data/training-mltable-folder
```

Comment:
For standalone CLI jobs, we have a readme on how to prepare the data before using it in CLI jobs. Since we are referring to those MLTable files from here, I think we should have a similar readme.md in each of these folders. 
The readme.md should explain how to prepare the data (upload image data and create jsonl files in appropriate folder).